apps/ios/Shared/Views/WelcomeView.swift

```diff
@@ -7,10 +7,15 @@
 
 import SwiftUI
 
+func isValidDisplayName(_ name: String) -> Bool {
+    return !(self.rangeOfCharacter(from: .whitespacesAndNewlines) != nil)
+}
```

Comment:
```suggestion
func invalidDisplayName(_ name: String) -> Bool {
    return self.rangeOfCharacter(from: .whitespacesAndNewlines) != nil
}
```

I also think self is not needed here?